### DISCUSSION TITLE
[Request] Suppresss javascript exceptions

### DISCUSSION THREAD

**Discussion by BrandonMayjeye** [Voltstro-Studios/UnityWebBrowser#360] [Requests]
### What platform are you experiencing this issue on?

Windows x64

### What version of UWB are you using?

2.2.2

### What Unity version are you running?

2022.3.48f1

### Describe what the issue you are experiencing is.

I would like to suggest the ability to suppress java script exceptions , as it currently stands development becomes tedious because the game is frozen each time theirs a minor java script error.

### Provide reproducible steps for this issue.

<script>
loadTransport()
</script>

running this in the browser will pause the unity debugger, because all exceptions from cef are treated as such , even if its only javascript errors.

### Any additional info you like to provide?

I know its possible to disable unity pause on exception , but this feature is handy for other errors.
whereas any page can technically have a javascript error.

**Comment by Voltstro** [comment]
Please open feature requests in Discussions, not as a bug report.

---

I don't see the point in doing this as it has nothing to really do with UWB, rather it is a config with your IDE. 

If you want to, you can can set [Logger](https://projects.voltstro.dev/UnityWebBrowser/latest/api/voltstrostudios.unitywebbrowser.core.webbrowserclient/#VoltstroStudios_UnityWebBrowser_Core_WebBrowserClient_Logger) to a custom [IWebBrowserLogger](https://projects.voltstro.dev/UnityWebBrowser/latest/api/voltstrostudios.unitywebbrowser.logging.iwebbrowserlogger/) that does not have LogError.
